### ISSUE TITLE
Rotation Axis [111], Grain Boundary Plane (1 -2 1) is not generating the structure

### ISSUE THREAD

**Issue by maaliso** [ksyang2013/aimsgb#15]
Hi everyone, 

I am trying to use the aimsgb python version to generate Rotation Axis [111], Grain Boundary Plane (1 -2 1) grain boundaries, but it is not working on the python interface. I tried the same with the website interface and it working and generating a structure. Is there any special thing about this specific arrangement or structure?

My code is:
```
atoms = Grain.from_file(f'uc_relaxed.POSCAR')
gb = GrainBoundary([1, 1, 1], 3, [1, -2, 1], atoms, uc_a=i+1, uc_b=i+1)
structure = Grain.stack_grains(gb.grain_a, gb.grain_b, direction=gb.direction)
structure.to(filename=f'{i}_{gb_name}.POSCAR')
```
When I try this code, I always get a unit cell in the output even though the same code is generating all other structures that I have used.

Thanks in advance for your help!!

**Comment by jic198** [comment]
Hello,

Your code looks good to me. Do you mind sharing your input file here? I will test it from my side.

**Comment by maaliso** [comment]
[uc_relaxed (copy).txt](https://github.com/ksyang2013/aimsgb/files/15270672/uc_relaxed.copy.txt)
Hi jic198, Thanks for your reply. 
This is the POSCAR file, where the unit cell is. I just changed the extension of the file so that it will be attached to the comment.

**Comment by jic198** [comment]
I used your POSCAR file and it generated the structure I expected. I used `uc_a=3, uc_b=3`. Is it not what you wanted?
[POSCAR.txt](https://github.com/ksyang2013/aimsgb/files/15274993/POSCAR.txt)

<img width="661" alt="image" src="https://github.com/ksyang2013/aimsgb/assets/18478036/115fe924-40e4-4698-a364-768ed736db4d">

**Comment by maaliso** [comment]
Yes, it is what I want. I see. Interesting. I got this too with the web interface, but I don't know why I don't get it from the python package of aimsgb. 
[code_for_gb_generation.txt](https://github.com/ksyang2013/aimsgb/files/15276270/code_for_gb_generation.txt)


I will attach the python file with I used right now to replicate what you did, with the output and everything and I didn't remove the mistakes even. If you have time and can check this, I would appreciate it. Otherwise, maybe I need to update the version of aimsgb that I have.

**Comment by jic198** [comment]
I didn't see any error in your code. Could you update aimsgb and try again?

**Comment by kaijiz** [comment]
Can i ask why you don't observe the structure from b direction or c direction but in b+c direction?
